.travis.yml

```diff
@@ -10,10 +15,12 @@ php:
 matrix:
     allow_failures:
         - php: hhvm
+    fast_finish: true
 
 before_script:
-  - ./bin/configure_test_env.sh
-  - composer self-update && composer install --dev --prefer-source --no-interaction
+  - echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini

```

Comment:
this should be skipped on HHVM (and PHP 7 but it is not yet enabled) as mongo.so is not available there